app/Http/Controllers/SeachController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Product;
+
+class SeachController extends Controller
+{
+    public function seach(Request $request)
+    {
+        $key = $request['key'];
+        $products = [];
+        foreach (Product::all() as $product) {
+            if (strpos($product->name, $key) > 0) {
```

Comment:
write new method for Product -> search()
use where('name', 'like', '%' . $keywork . '%')